server/controllers/recommendations/postRecommendationForm.test.ts

```diff
@@ -39,19 +40,22 @@ describe('postRecommendationForm', () => {
       method: 'POST',
       params: { recommendationId, pageId },
       body: {
+        recallType: 'STANDARD',
         crn,
       },
     })
     await postRecommendationForm(req, res)
     expect(res.redirect).toHaveBeenCalledWith(303, currentPageUrl)
     expect(req.session.errors).toEqual([
       {
-        href: '#recallType',
-        name: 'recallType',
-        text: 'Select a recommendation',
-        errorId: 'noRecallTypeSelected',
+        errorId: 'missingRecallTypeDetail',
+        fieldText: 'Enter more detail',
```

Comment:
Is this message correct? The ACs in MRD-460 suggest something different. Can you update the ticket if appropriate?

Reply:
yep will update ticket. As mentioned, this needs approving by the content designer anyway